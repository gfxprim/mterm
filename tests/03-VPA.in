10 25
\e[H\e[2J\e[24da
