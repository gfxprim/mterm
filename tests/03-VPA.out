 ----------
|          |
|          |
|          |
|          |
|          |
|          |
|          |
|          |
|          |
|          |
|          |
|          |
|          |
|          |
|          |
|          |
|          |
|          |
|          |
|          |
|          |
|          |
|          |
|a         |
|          |
 ----------
size 25x10 cursor 23x1
